patches/components-invalidation-impl-fcm_network_handler.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/invalidation/impl/fcm_network_handler.cc b/components/invalidation/impl/fcm_network_handler.cc
+index cbf7f786cb7f..aff1bd9cf428 100644
+--- a/components/invalidation/impl/fcm_network_handler.cc
++++ b/components/invalidation/impl/fcm_network_handler.cc
+@@ -86,6 +86,7 @@ FCMNetworkHandler::~FCMNetworkHandler() {
+ }
+ 
+ void FCMNetworkHandler::StartListening() {
++  return; // feature disabled in Brave
```

Comment:
I think this patch can be replaced with a chromium_src override on FCMInvalidationService and a subclass of FCMNetworkHandler (StartListening is a virtual method)